src/pkg/packager/deploy.go

```diff
@@ -109,7 +109,7 @@ func (p *Packager) deployComponents() (deployedComponents []types.DeployedCompon
 		config.SetDeployingComponents(deployedComponents)
 	}
 
-	config.ClearDeployingComponents()
+	// config.ClearDeployingComponents()
```

Comment:
Make sure we aren't leaving commented code

Reply:
Fixed